testbed/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@adobe/node-httptransfer-cli",
+  "version": "1.0.0",
+  "main": "index.js",
+  "description": "Efficient File Transfer Module for NodeJS CLI",
+  "author": {
+    "name": "Adobe Systems Inc."
+  },
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/adobe/node-httptransfer.git"
+  },
+  "dependencies": {
+    "@azure/storage-blob": "^10.5.0",
+    "@nui/node-httptransfer": "file:..",
```

Comment:
this should be the open sourced dependency

Reply:
Actually this line must be removed. Can't depend on itself.

Reply:
I removed the dep, to require the file directly (there is another module hidden inside node-httptransfer, which is the cli, `testbed`).